# The patch replaces the Sundials from OpenModelica 3dParty directory by the Sundials from EasyBuild
# author: Jiri Furst <jiri.furst@gmail.com>
--- OMCompiler/3rdParty/sundials-5.4.0/config/SundialsFortran.cmake.orig	2021-10-18 19:34:37.946396658 +0200
+++ OMCompiler/3rdParty/sundials-5.4.0/config/SundialsFortran.cmake	2021-10-18 19:33:44.030397643 +0200
@@ -24,15 +24,15 @@
 set(ENV_FCFLAGS "$ENV{FCFLAGS}")
 
 # check if environment variables are used and CMAKE_Fortran_FLAGS is not
-if ((NOT "${ENV_FFLAGS}" STREQUAL "") AND (NOT "${ENV_FCFLAGS}" STREQUAL "")
-    AND ("${CMAKE_Fortran_FLAGS}" STREQUAL ""))
-
-  # check if environment variables are equal
-  if (NOT "${ENV_FFLAGS}" STREQUAL "${ENV_FCFLAGS}")
-    print_error("FFLAGS='${ENV_FFLAGS}' and FCFLAGS='${ENV_FCFLAGS}' are both set but are not equal.")
-  endif()
-
-endif()
+#if ((NOT "${ENV_FFLAGS}" STREQUAL "") AND (NOT "${ENV_FCFLAGS}" STREQUAL "")
+#    AND ("${CMAKE_Fortran_FLAGS}" STREQUAL ""))
+#
+#  # check if environment variables are equal
+#  if (NOT "${ENV_FFLAGS}" STREQUAL "${ENV_FCFLAGS}")
+#    print_error("FFLAGS='${ENV_FFLAGS}' and FCFLAGS='${ENV_FCFLAGS}' are both set but are not equal.")
+#  endif()
+#
+#endif()
 
 # Enable Fortran
 enable_language(Fortran)
